lib/hotkeys.ts

```diff
@@ -0,0 +1,29 @@
+const MODIFIERS = {
+  alt: 'altKey',
+  control: 'ctrlKey',
+  meta: 'metaKey',
+  shift: 'shiftKey'
+} as const
+
+type Modifier = keyof typeof MODIFIERS
+
+function compareHotkey(hotkey: string, e: KeyboardEvent): boolean {
+  let keys = hotkey.split('+')
+
+  for (let key of keys) {
+    if (key in MODIFIERS) {
+      let modifier = MODIFIERS[key as Modifier]
+      if (!e[modifier]) {
+        return false
```

Comment:
We need to not call `ctrl+z` hotkey on `ctrl+alt+z` press.

You can check a little different approach in `keyux` library.

https://github.com/ai/keyux/blob/main/hotkey.js#L66

Reply:
yes, it's better know. and object is no longer needed